mobile/src/lib/reporter.js

```diff
@@ -7,7 +7,8 @@ export const ANALYTICS_URL = `https://${__SENTRY_TOKEN__}@sentry.cozycloud.cc/2`
 
 export function getAnalyticsConfiguration () {
   return {
-    shouldSendCallback: () => getState().mobile.settings.analytics
+    shouldSendCallback: () => getState().mobile.settings.analytics,
+    environment: __DEVMODE__ ? 'development' : 'production'
```

Comment:
I forgot to explain: within the same project, you can create multiple DSN (access keys). I though it permits to discriminate sentry's message, but it doesn't. So I can remove one of the DSN as it is not useful, nor used.